### DISCUSSION TITLE
Devs Need OldWorld PowerBook?

### DISCUSSION THREAD

**Discussion by hilga007** [kth5/archpower#51] [Ideas]
Hi there! 

I have an elderly 12" PowerBook G3 Wallstreet laptop. Any devs want it for testing "for the lulz" of modern Arch Linux on an elderly computer? (Prefer if you are in USA, Canada, Mexico, Caribbean, or EU nations for cheaper shipping rates) 

_Also documents process for Wiki so is not only for lulz_

(Other use case: using legacy Apple-specific ports to communicate with old audio or scientific equipment via Arch/AUR in future)

**Comment by kth5** [comment]
Hi @hilga007,

while the prospect of supporting such old systems is intriguing to me as a nerd, as  professional I must say usefulness of this would be negigible though.

Booting an old world Mac like this requires booting into System 7, or MacOS 8/9 first to run BootX which is the only bootloader I know of supporting OldWorld Macs and it has been unsupported for years.

https://www.penguinppc.org/bootloaders/bootx/

Once BootX is installed on a host System/MacOS - boot floppy usually enough, it should be as easy as loading up the kernel and initramfs like yaboot/grub would.

Be advised, you need an absolute minimum of about 180MB RAM if you want to boot the Arch POWER ISO, of you'll have to build/provide your own installation environment.
https://github.com/kth5/archpower/wiki/Powerpc-%7C-Minimum-Requirements

**Reply by iamhumanipromise** [reply]
I could see using an old G3 as a "Pi Hole" on my dad's 10mb dsl connection and act as print server for an apple laser printer. 

Old hardware can still have a use. 

Willing to learn how to build packages to have altivec-free packages for G3, if you or anyone is willing to teach

**Comment by kth5** [comment]
All packages for Arch POWER `powerpc` do not enable altivec by default. Only when the build system requires it there are expceptios in the `PKGBUILD`.  Some things won't build without altivec at all and need custom fixes in form of patches. I'm thinking `ffmpeg` and `mplayer` etc. Though media playback on a 200-300Mhz G3 is probably not a priority.
